Makefile

```
@@ -5,7 +5,8 @@
 ## at 'http://localhost' with password 'secretpassword'
 .PHONY: help setup up down clean
 
-BBOX := 25,20,35,30
+BBOX := 14.6,-36.3,35.9,-20.7
+INDEX_LIMIT := 10
```

I think this should be bigger, say, `1,000`.
```suggestion
INDEX_LIMIT := 1000
```

---

I shall test the time this takes, and commit

---

For 10 datasets it takes 5 minutes
For 100 datasets it takes 12 minutes
for 1000 datasets it takes 1.5 hours